plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <string.h>
+#include <libmm-glib.h>
+
+#include "fu-plugin-vfuncs.h"
+
+#include "fu-mm-device.h"
+#include "fu-mm-utils.h"
+
+/* amount of time to wait for ports of the same device	being exposed by kernel */
+#define FU_MM_UDEV_DEVICE_PORTS_TIMEOUT		3	/* s */
+
+typedef struct FuPluginMmInhibitedDevice FuPluginMmInhibitedDevice;
+
+struct FuPluginData {
+	MMManager	*manager;
+	gboolean	manager_ready;
+
+	/* when a device is inhibited from MM, we store all relevant details
+	 * ourselves to recreate a functional device object even without MM
+	 */
+	FuPluginMmInhibitedDevice *inhibited;
+};
+
+struct FuPluginMmInhibitedDevice {
+	gchar	 *inhibited_uid;
```

Comment:
one more tab level of indenting please

Reply:
I pushed a change, not sure if it's exactly what you expected in this one